www-client/chromium/chromium-95.0.4638.17.ebuild

```
@@ -822,6 +880,90 @@ src_compile() {
 	use suid && eninja -C out/Release chrome_sandbox
 
 	pax-mark m out/Release/chrome
+}
+
+# This function is called from virtx, and must always return so that Xvfb
+# session isn't left running. If we return 1, virtx will call die().
+chromium_profile() {
+	local case_name cur prev prev_arg
+
+	cur="${T}/cur.profdata"
+	prev="${T}/prev.profdata"
+
+	einfo "Profiling for PGO"
+
+	pushd "${WORKDIR}/chromium-profiler-"* >/dev/null || return 1
+
+	# Remove old profdata in case profiling was interrupted.
+	rm -rf profdata "$cur" "$prev"
+
+	# We profile each case separately and then merge the resulting profile
+	# data with the previous data incrementally into one file.
+	# This is done so that we can remove all but the one resulting file,
+	# since profile data take up gigabytes of space.
+	for case_name in $(./chromium_profiler.py --list-cases); do
+		einfo "Profiling case ${case_name}"
+
+		if ! "${EPYTHON}" ./chromium_profiler.py \
+		     --chrome-executable "${S}/out/Release/chrome" \
+		     --chromedriver-executable "${S}/out/Release/chromedriver.unstripped" \
+		     --add-arg no-sandbox --add-arg disable-dev-shm-usage \
+		     --case "${case_name}"; then
+			eerror "Profiling case ${case_name} failed"
+			return 1
+		fi
+
+		if [[ -e "$cur" ]]; then
```

```suggestion
		if [[ -e "${cur}" ]]; then
```

ditto other cases, see [here](https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101)